### PR TITLE
Lexicographically largest palindrome subsequence in cpp

### LINKED ISSUE #3007: Lexicographically Largest Palindromic Subsequence
## 💥 Proposal

(A clear and concise description of what the proposal is.)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

(Write your answer here.)
Hey I am Participant of GSSOC'21. And I have read all the Contributing Guidelines.
I want to propose a program which take string of any length as input and in output we get lexicographically largest palindrome string with its length. 
For Example:  Input = bassoon   Output = ss 2. 
@ankitaggarwal23  Please assign me this issue.

### PR DESCRIPTION
**Pull request by sanskruti-shahu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes, I have read the guidelines.

### Description

Added source code for finding Lexicographically largest palindrome subsequence

### Checklist

- [ Yes ] I've read the contribution guidelines.
- [ Yes ] I've checked the issue list before deciding what to submit.
- [ Yes ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #3007

### CODE REVIEW

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@
 - [Moore's Voting Algorithm to find the majority element](cp/moore_voting.cpp)
 - [Ticket Auction](cp/tkt_auction.cpp)
 - [Replace Pi using recursion](cp/replace_pi.cpp)
+- [Lexicographically largest palindrome susequence](cp/Lexicographyically_largest_palindrome_subsequence.cpp)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/README.md]: Typo Error:`susequence` Kindly fix this.

**Review by ankitaggarwal23 — COMMENTED** [review]
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top
- Use proper code Format 
- Remove extra lines of code and spaces
- Add a new line at the EOF

**Comment by sanskruti-shahu** [comment]
Hey!! I updates changes.
@ankitaggarwal23  Please review.

File: C-Plus-Plus/README.md (hunk 1 above)
Comment (gat786) [reply C-Plus-Plus/README.md]: that's `sus`

File: C-Plus-Plus/cp/Lexicographyically_largest_palindrome_subsequence.cpp [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Here, Characters in the strings are compared according to their ASCII codes and the one with largest ASCII code is lexicographically largest. 
+And output contain frequency and all the reoccurrence of that character in string S in a single line.
+*/     
+#include<iostream>
+#include<string>
+using namespace std;
+
+int main() {
+    string S, k, a, m;
+    int i, j, c, max;
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/Lexicographyically_largest_palindrome_subsequence.cpp]: - Use Good variables name.
- Be clear and concise.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Shift the logic to another function and use main for driver code.

**Comment by sanskruti-shahu** [comment]
I have updated it again. Please review @ankitaggarwal23

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**